integrations/aws/.port/resources/port-app-config.yml

```diff
@@ -0,0 +1,47 @@
+resources:
+  - kind: AWS::Organizations::Account
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .Id
+          title: .Name
+          blueprint: '"awsAccount"'
+          properties:
+            arn: .Arn
+            email: .Email
+            status: .Status
+            joined_method: .JoinedMethod
+            joined_timestamp: .JoinedTimestamp
+
+  - kind: cloudResource
+    selector:
+      query: 'true'
+      resourceKinds:
+        - AWS::Lambda::Function
+        - AWS::RDS::DBInstance
+        - AWS::S3::Bucket
+        - AWS::IAM::User
+        - AWS::ECS::Cluster
+        - AWS::ECS::Service
+        - AWS::Logs::LogGroup
+        - AWS::DynamoDB::Table
+        - AWS::SQS::Queue
+        - AWS::SNS::Topic
+        - AWS::Cognito::IdentityPool
+        - AWS::CloudFormation::Stack
```

Comment:
I don't think we need that much of resources as part of the first installation. wdyt?